test/md-to-react.spec.js

```diff
@@ -30,4 +30,10 @@ ${codeBlockText}
     const wrapper = mount(mdToReact(markdown));
     assert.ok(wrapper.find('code').hasClass('hljs') && wrapper.find('code').hasClass('css'));
   });
+
+  it('should not guess the language when autodetect is false', () => {
+    const markdown = '```.test { height: 1em; }```';
+    const wrapper = mount(mdToReact(markdown), { autodetect: false });
+    assert.ok(wrapper.find('code').hasClass('hljs') && wrapper.find('code').hasClass('css'));
```

Comment:
I could be wrong about this... but isn't this test case the same as the one on line 31 for when it *should* guess the language? Shouldn't this be different?

Reply:
You're right, that test was broken. I fixed it.